src/utils/reportHelpers.ts

```diff
@@ -43,6 +50,83 @@ const getMonth = (dateStr: string): number => {
+      TAE: finances[1].frameBudget,
+      TSE1: finances[2].frameBudget,
+      TSE2: finances[3].frameBudget,
+      initial1: finances[4].frameBudget,
+      initial2: finances[5].frameBudget,
+      initial3: finances[6].frameBudget,
+      initial4: finances[7].frameBudget,
+      initial5: finances[8].frameBudget,
+      initial6: finances[9].frameBudget,
+      initial7: finances[10].frameBudget,
+    }
+  }
+  return {
+    plannedCostForecast: finances[0].plannedBudget,
+    plannedTAE: finances[1].plannedBudget,
+    plannedTSE1: finances[2].plannedBudget,
+    plannedTSE2: finances[3].plannedBudget,
+    plannedInitial1: finances[4].plannedBudget,
+    plannedInitial2: finances[5].plannedBudget,
+    plannedInitial3: finances[6].plannedBudget,
+    plannedInitial4: finances[7].plannedBudget,
+    plannedInitial5: finances[8].plannedBudget,
+    plannedInitial6: finances[9].plannedBudget,
+    plannedInitial7: finances[10].plannedBudget,
+  }
+};
+
+const sumFinances = (classItem: IPlanningRow, category: Categories, totalsParam?: ITotals) => {
+  const initialTotals =  {
+    plannedCostForecast: 0,
+    plannedTAE: 0,
+    plannedTSE1: 0,
+    plannedTSE2: 0,
+    plannedInitial1: 0,
+    plannedInitial2: 0,
+    plannedInitial3: 0,
+    plannedInitial4: 0,
+    plannedInitial5: 0,
+    plannedInitial6: 0,
+    plannedInitial7: 0,
+  };
+  let totals = totalsParam || initialTotals;
+
+  classItem.projectRows.forEach((obj) => {
+    if (obj.category?.value === category) {
+      totals.plannedCostForecast += Number(obj.finances?.budgetProposalCurrentYearPlus0?.replace(/\s/g, ''));
+      totals.plannedTAE += Number(obj.finances?.budgetProposalCurrentYearPlus1?.replace(/\s/g, ''));
+      totals.plannedTSE1 += Number(obj.finances?.budgetProposalCurrentYearPlus2?.replace(/\s/g, ''));
+      totals.plannedTSE2 += Number(obj.finances?.preliminaryCurrentYearPlus3?.replace(/\s/g, ''));
+      totals.plannedInitial1 += Number(obj.finances?.preliminaryCurrentYearPlus4?.replace(/\s/g, ''));
+      totals.plannedInitial2 += Number(obj.finances?.preliminaryCurrentYearPlus5?.replace(/\s/g, ''));
+      totals.plannedInitial3 += Number(obj.finances?.preliminaryCurrentYearPlus6?.replace(/\s/g, ''));
+      totals.plannedInitial4 += Number(obj.finances?.preliminaryCurrentYearPlus7?.replace(/\s/g, ''));
+      totals.plannedInitial5 += Number(obj.finances?.preliminaryCurrentYearPlus8?.replace(/\s/g, ''));
+      totals.plannedInitial6 += Number(obj.finances?.preliminaryCurrentYearPlus9?.replace(/\s/g, ''));
+      totals.plannedInitial7 += Number(obj.finances?.preliminaryCurrentYearPlus10?.replace(/\s/g, ''));
+    }
+  });
+
+  if (classItem.children) {
+    classItem.children.forEach((child) => {
+      totals = sumFinances(child, category, totals);
+    });
+  }
+
+  return totals;
+}
+
+const getPlannedBudgetsByCategories = (classItem: IPlanningRow, category: Categories) => {
+  const sums = sumFinances(classItem, category);
+  return sums;
+}
```

Comment:
This function only returns the result of another function. Maybe it's not needed?